fragments/k8s/core/bundle.yaml

```diff
@@ -9,9 +9,8 @@ applications:
     to:
       - "0"
     options:
-      channel: 1.23/edge
+      channel: 1.24/edge
     expose: true
-    constraints: "cores=2 mem=4G root-disk=16G"
```

Comment:
removed the charm constraint since these are defined in the `machines` stanza below.

Reply:
Will this drop the application constraints? I.e. if you run

```
juju get-constraints kubernetes-control-plane
```

after deployment, what will it show?

I agree that these constraints are not used at deploy time, but I'm concerned they may still be relevant when `juju add-unit` is called later in the cluster lifecycle.

Reply:
aaahhhh... excellent point.  you're right, without the app constraints, add-unit uses the cloud default.